ipa-core/src/net/server/handlers/query/metrics.rs

```diff
@@ -0,0 +1,28 @@
+use axum::{routing::get, Extension, Router};
+use hyper::StatusCode;
+
+use crate::{
+    helpers::{routing::RouteId, BodyStream},
+    net::{
+        http_serde::{self},
+        Error, MpcHttpTransport,
+    },
+};
+
+/// Takes details from the HTTP request and creates a `[TransportCommand]::CreateQuery` that is sent
+/// to the [`HttpTransport`].
+async fn handler(transport: Extension<MpcHttpTransport>) -> Result<Vec<u8>, Error> {
```

Comment:
I'd encourage to write some tests here as it is often hard to figure out why HTTP server is not responding or responds with errors. There are some examples in this folder that exercise happy path and error path